cmd/manager/main.go

```diff
@@ -100,7 +106,7 @@ func main() {
 	}
 
 	cl := mgr.GetClient()
-	catalogClient := catalogclient.New(cl)
+	catalogClient := catalogclient.New(cl, cache.NewFilesystemCache(cachePath, http.DefaultClient))
```

Comment:
https://medium.com/@nate510/don-t-use-go-s-default-http-client-4804cb19f779

Reply:
TL;DR: we need to set some sensible timeouts on the client.

Reply:
TIL

Reply:
Updated in the latest push (https://github.com/operator-framework/operator-controller/pull/411/commits/1427f7950c3e0c3b2fa56916795b133cf149a4c7)